cmd/crl-checker/main.go

```diff
@@ -100,7 +100,14 @@ func main() {
 
 		totalBytes += len(crl.Raw)
 
-		err = checker.Validate(crl, issuer, ageLimit)
+		zcrl, err := crl_x509.ParseRevocationList(crl.Raw)
+		if err != nil {
+			errCount += 1
+			logger.Errf("checking CRL %q failed: %s", u, err)
```

Comment:
Nit: This log line is identical to line 113 which might be confusing if we ever had to debug this during an incident.
```suggestion
			logger.Errf("parsing CRL %q failed: %s", u, err)
```